pac4j-oauth/src/main/java/org/pac4j/oauth/profile/facebook/FacebookProfile.java

```diff
@@ -57,127 +57,138 @@ public String getLocation() {
         }
         return null;
     }
-    
+
     public String getMiddleName() {
         return (String) getAttribute(FacebookAttributesDefinition.MIDDLE_NAME);
     }
-    
+
     public List<FacebookObject> getLanguages() {
         return (List<FacebookObject>) getAttribute(FacebookAttributesDefinition.LANGUAGES);
     }
-    
+
     public String getThirdPartyId() {
         return (String) getAttribute(FacebookAttributesDefinition.THIRD_PARTY_ID);
     }
-    
+
     public Integer getTimezone() {
         return (Integer) getAttribute(FacebookAttributesDefinition.TIMEZONE);
     }
-    
+
     public Date getUpdateTime() {
         return (Date) getAttribute(FacebookAttributesDefinition.UPDATED_TIME);
     }
-    
+
     public Boolean getVerified() {
         return (Boolean) getAttribute(FacebookAttributesDefinition.VERIFIED);
     }
-    
+
+	/**
```

Comment:
Master is at version 2.0.0, which is a major one with potential breaking changes.

So you can safely remove this method.